src/com/aca/ArmineArzumanyan/homeWork_1/manager/GadgetStoreManager.java

```diff
@@ -0,0 +1,65 @@
+package com.aca.ArmineArzumanyan.homeWork_1.manager;
+
+import com.aca.ArmineArzumanyan.homeWork_1.items.gadget.Gadget;
+import com.aca.ArmineArzumanyan.homeWork_1.repository.GadgetsRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GadgetStoreManager implements Manager<Gadget> {
```

Comment:
It is very nice that you know about generics Armine jan, but I have a question, do you use this Manager template for another class also? So do you have a manager which is responsible for another type of item? No, we don't! So why complicate our code?   please read about YAGNI and KISS principles from the "clean code" theory

Reply:
I think we will expend our app, and I will need manager which is responsible for another type of item.